Assets/PatchKit Patcher/Scripts/App.cs

```diff
@@ -33,6 +33,13 @@ public class App
 
         private readonly int _overrideLatestVersionId;
 
+        public enum InstallStatus
+        {
+            NotInstalled = 0,
+            MaybeInstalled = 1,
```

Comment:
I called it MaybeInstalled because this status could mean that the installation has been started but not finished, or it has been installed and then some files got deleted.

Calling it PartlyInstalled would imply the first scenario.

Reply:
Thinking about it some more, I could simply call it `Other`. @genail ?

Reply:
`Other` would be fine too.

Reply:
Sounds good